src/Tickets/Commerce/Gateways/Stripe/Assets.php

```diff
@@ -56,10 +56,11 @@ public function register() {
 				'tribe-tickets-commerce-base-gateway-checkout-js',
 				'tribe-tickets-commerce-base-gateway-checkout-toggler',
 			],
-			'wp_enqueue_scripts',
+			null,
```

Comment:
@rafsuntaskin The same comment here, why is that we're adding the `wp_enqueue_scripts`? Should be enough if it's part of the group that we're loading on checkout (`tribe-tickets-commerce-checkout`)